webextensions/background/handle-misc.js

```diff
@@ -26,15 +26,10 @@ import * as Background from './background.js';
 import * as TabsGroup from './tabs-group.js';
 import * as Tree from './tree.js';
 import * as Commands from './commands.js';
-import * as HandleTabMultiselect from './handle-tab-multiselect.js';
```

Comment:
This module still need to be loaded from anywhere on the background page (for example background/index.js or background/background.js) to multiselect collapsed tree.

Reply:
Sorry wasn't really understanding how javascript import works. In any case you appear to have already been loading this module in [`background/index.js`](https://github.com/piroor/treestyletab/commit/7d5a8b06d7adf894ba6d8fac0488c6c2f74532b0#diff-9f54d9eadb1d83ec226914ccb3adb65aR18). I just tested that multiselect collapsed tree (where collapsed tabs are automatically selected as well) still works.

Reply:
Oh, you right. I missed that it was already loaded from index.js.